operator/templates/generate-cqlshrc-sh.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Name }}-generate-cqlshrc-sh
+  namespace: {{ .Namespace }}
+data:
+  generate-cqlshrc.sh: |
```

Comment:
Could this be a configmap of `cqlshrc` itself instead of a bash script that generates it?

Reply:
That is what I was going to but, I made a script so that in future if additional configuration is needed for cqlsh, it can be in one script; like adding custom config containing additional config for cqlsh ref: https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlshUsingCqlshrc.html

Reply:
What's the difference in having a templated bash script that outputs a file from a templated file?

Reply:
like here what we do for kafka custom properties taken from a user provided CM https://github.com/kudobuilder/operators/blob/ebaf9e6dba463aa4ff8ec7b4e537c7fdc8c3f629/repository/kafka/operator/templates/bootstrap.yaml#L175

Reply:
If this is not required for cqlshrc file then I can revert back to the usual way for mounting a CM and cp it

Reply:
@mpereira as discussed we are keeping the script based approach as POD_NAME env variable is used in the script to create the hostname